package.json

```diff
@@ -39,9 +39,10 @@
     "js-yaml": "~3.2.2",
     "node-txstatsd": "~0.1.5",
     "node-uuid": "git+https://github.com/gwicke/node-uuid#master",
-    "preq": "^0.4.0",
+    "preq": "~0.4.0",
```

Comment:
IMHO, better to integrate #256 first and then just add the `jsonwebtoken` dep, since that PR cleans up and refreshes deps and their versions.